test/src/test/groovy/org/openremote/test/protocol/mqtt/MQTTClientProtocolTest.groovy

```diff
@@ -70,13 +79,47 @@ import java.security.PrivateKey
 import java.security.cert.Certificate
 import java.security.cert.CertificateFactory
 import java.security.cert.X509Certificate
+import java.util.concurrent.TimeUnit
 
 import static org.openremote.manager.mqtt.MQTTBrokerService.getConnectionIDString
+import static org.openremote.model.Constants.MASTER_REALM
 import static org.openremote.model.value.MetaItemType.AGENT_LINK
+import static org.openremote.model.value.ValueType.BOOLEAN
 import static org.openremote.model.value.ValueType.NUMBER
 
 class MQTTClientProtocolTest extends Specification implements ManagerContainerTrait {
 
+    @Shared
+    int mqttBrokerPort
+
+    @Shared
+    Server mqttBroker
```

Comment:
If you're running the whole manager then you can just connect to our own broker

Reply:
I've made the change and replaced the Moquette test broker with the internal OR broker.